cc/transport/grpc_streaming_transport.cc

```diff
@@ -59,11 +63,11 @@ absl::StatusOr<AttestationBundle> GrpcStreamingTransport::GetEvidence() {
   }
 }
 
-absl::StatusOr<std::string> GrpcStreamingTransport::Invoke(absl::string_view request_bytes) {
+absl::StatusOr<EncryptedResponse> GrpcStreamingTransport::Invoke(
+    const EncryptedRequest& encrypted_request) {
   // Create request.
   RequestWrapper request;
-  InvokeRequest* invoke_request = request.mutable_invoke_request();
-  invoke_request->set_encrypted_body(request_bytes);
+  *request.mutable_invoke_request()->mutable_encrypted_request() = encrypted_request;
```

Comment:
More of an open-ended question here, but how large do we expect requests to be? Is copying ```encrypted_request``` here acceptable, or should we instead have the invoke function use the direct value of ```encrypted_request```?

Reply:
We don't limit the size of a message, so technically it could be max proto size (which is 2Mb I believe).

Though `Invoke` is async, so we probably shouldn't rely on the argument lifetime to use it as a pointer.